test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -617,14 +617,34 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 					HighPowerConsumption:  pointer.Bool(true),
 					RealTime:              pointer.Bool(true),
 				}
-				EventuallyWithOffset(1, func() string {
-					err := testclient.ControlPlaneClient.Update(context.TODO(), profile)
-					if err != nil {
-						statusErr, _ := err.(*errors.StatusError)
-						return statusErr.Status().Message
-					}
-					return fmt.Sprint("Profile applied successfully")
-				}, time.Minute, 5*time.Second).Should(ContainSubstring("HighPowerConsumption and PerPodPowerManagement can not be both enabled"))
+				if hypershift.IsHypershiftCluster() {
+					hostedClusterName, err := hypershift.GetHostedClusterName()
+					np, err := nodepools.GetByClusterName(ctx, testclient.ControlPlaneClient, hostedClusterName)
+					Expect(err).ToNot(HaveOccurred())
+					EventuallyWithOffset(1, func() string {
+						err := testclient.ControlPlaneClient.Update(context.TODO(), profile)
```

Comment:
The Update should be called out side the `EventuallyWithOffset` 
```
Expect(testclient.ControlPlaneClient.Update(context.TODO(), profile)).To(Succeed())
EventuallyWithOffset(1, func() string {
...
```

Reply:
fixed in the latest commit